### PR TITLE
LPS-147452 | Master

### PR DESCRIPTION
**Pull request by Jeremy-Ch**
https://issues.liferay.com/browse/LPS-147452

### CODE REVIEW

**Comment by Jeremy-Ch** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DMCache.testcase [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+			sourceFileName = "Commerce_Blue.jpg");
+
+		Navigator.gotoPage(pageName = "Documents and Media Page");
+
+		DMNavigator.gotoDocumentPG(dmDocumentTitle = "DM Document Title");
+
+		var dmDocumentURL = selenium.getAttribute("//img[contains(@class,'preview-file-image') and contains(@src,'Commerce_Blue.jpg')]@src");
+
+		var curl = '''${dmDocumentURL} -I''';
+		var responseHeader = JSONCurlUtil.post("${curl}");
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DMCache.testcase:65]: @Jeremy-Ch can you make this into a macro?

Comment (austinchiang) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DMCache.testcase:65]: Same with the other tests

**Comment by Jeremy-Ch** [comment]
@austinchiang See https://github.com/austinchiang/liferay-portal/pull/1267

**Pull request closed without merging**